packages/libs/deer/Makefile.toml

```diff
@@ -0,0 +1,41 @@
+extend = { path = "../../../.github/scripts/rust/Makefile.toml" }
+
+[env]
+CARGO_CLIPPY_HACK_FLAGS = "--workspace --feature-powerset --exclude-features default"
+CARGO_TEST_HACK_FLAGS = "--workspace --feature-powerset --exclude-features default"
+CARGO_RUSTDOC_HACK_FLAGS = ""
+CARGO_DOC_HACK_FLAGS = ""
+
+# deer does not have any unsafe code, therefore we do not need to test for miri
```

Comment:
Do you want to add a lint to forbid unsafe code? (If you haven't done it already)

Reply:
#1163 and [47db38a](https://github.com/hashintel/hash/pull/1164/commits/47db38aaa5b7df310810e818dc777c672aed6ffc) add this.